src/scene/immediate/immediate.js

```diff
@@ -231,19 +233,36 @@ class Immediate {
         batch.addLines(this.cubePositions, color);
     }
 
-    renderWireSphere(center, radius, color, options) {
+    renderWireBoundingBox(boundingBox, color, options) {
+        const min = boundingBox.getMin();
+        const max = boundingBox.getMax();
+        const cubeWorldPos = [
+            new Vec3(min.x, min.y, min.z), new Vec3(min.x, max.y, min.z), new Vec3(max.x, max.y, min.z), new Vec3(max.x, min.y, min.z),
```

Comment:
This does allocations on each render, which would be best to be avoided.

Reply:
yep, was planning to share the array with the function above .. all maybe remove it and share array with the spere and build flat float arrays

Reply:
and done, thanks!